docs/source/explanation/singleuser.md

```diff
@@ -0,0 +1,108 @@
+(singleuser)=
+
+# The JupyterHub single-user server
+
+When a user logs into JupyterHub, they get a 'server', which we usually call the **single-user server**, because it's a server that's meant for a single JupyterHub user.
+Each JupyterHub user gets a different one (or more than one!).
+
+A single-user server is a process running somewhere that is:
+
+1. accessible over http[s],
+2. authenticated via JupyterHub OAuth,
+3. started by a [Spawner](spawners), and
+4. 'owned' by a single JupyterHub user
+
+(singleuser-auth)=
+
+## Single-user server authentication
+
+Implementation-wise, JupyterHub single-user servers are a special-case of [](services),
+and as such use the same (oauth2) authentication mechanism (more on oauth in JupyterHub at [](oauth)).
+This is primarily implemented in the {class}`~.HubOAuth` class.
+
+The default single-user command, `jupyterhub-singleuser`, launches `jupyter-server`, the same program used when you run `jupyter lab` on your laptop.
```

Comment:
Maybe a brief expansion of "the default single-user command" to "the Spawner's default single-user server startup command" or similar.